test/support/tests_for_national_applicability.rb

```diff
@@ -21,6 +21,72 @@ module TestsForNationalApplicability
       assert_equal "http://www.scotland.com/", scotland_inapplicability.alternative_url
     end
 
+    test "create should create a new edition with all_nation_applicability including all nations" do
+      create(:government)
+      attributes = attributes_for_edition
+
+      expected_national_applicability = {
+        england: {
+          label: "England",
+          applicable: true,
+        },
+        northern_ireland: {
+          label: "Northern Ireland",
+          applicable: true,
+        },
+        scotland: {
+          label: "Scotland",
+          applicable: true,
+        },
+        wales: {
+          label: "Wales",
+          applicable: true,
+        },
+      }
```

Comment:
Since `expected_national_applicability` is exactly the same for both tests, can we pull it out and have it only once? It would avoid repetition and I think it would make the tests more readable.

Reply:
good idea